path.c

```diff
@@ -1270,7 +1270,7 @@ int is_mount_point_via_stat(struct strbuf *path)
 	}
 	current_dev = st.st_dev;
 
-	/* Now look at the current directoru */
+	/* Now look at the parent directory */
```

Comment:
Could you please squash that into the faulty commit instead of making it a follow-up commit?